README.md

```diff
@@ -115,6 +115,7 @@ Keep-Alive: timeout=5
 | unleashInstanceId         | `UNLEASH_INSTANCE_ID`            | `generated`     |    no    | Unleash instance id to used when registering with Unleash                                                                                                                                                                                                                                                                                                                                                                                                                |
 | refreshInterval           | `UNLEASH_FETCH_INTERVAL`         | 5000            |    no    | How often the proxy should query Unleash for updates, defined in ms.                                                                                                                                                                                                                                                                                                                                                                                                     |
 | metricsInterval           | `UNLEASH_METRICS_INTERVAL`       | 30000           |    no    | How often the proxy should send usage metrics back to Unleash, defined in ms.                                                                                                                                                                                                                                                                                                                                                                                            |
+| metricsJitter             | `UNLEASH_METRICS_JITTER`         | 0               |    no    | Adds jitter to the metrics interval to avoid multiple instances sending metrics at the same time, defined in ms.                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Was just about to ask for an update to the README. Refreshed the page and suddenly it appears! Magic